Database/Postgres/Controllers/controllers.js

```diff
@@ -0,0 +1,64 @@
+
+const models = require('../Models/models.js')
+exports.getAllProducts = async (req, res) => {
+    console.log("made it to getAllProducts")
+    console.log("page and count: ", req.body.page, req.body.count);
+    const page = req.body.page
```

Comment:
I think page & body would technically be query parameters so they should be referenced like req.query.page and req.query.count. The route would look like this /qa/products?count=1&page=1. I'll still approve since the main focus of the project is stress/load testing and scaling. But this wouldn't mesh with the front end if you end up wanting to join them.